google/cloud/CMakeLists.txt

```diff
@@ -303,9 +303,9 @@ set(GOOGLE_CLOUD_CPP_PC_DESCRIPTION
 # libraries, so we need to explicitly include them here. Here we include the
 # runtime deps of the absl::optional target (http://github.com/abseil/abseil-cpp
 # /blob/4a851046a0102cd986a5714a1af8deef28a544c4/absl/types/CMakeLists.txt#L164)
-set(GOOGLE_CLOUD_CPP_PC_LIBS
-    "-lgoogle_cloud_cpp_common -labsl_time -labsl_time_zone -labsl_bad_optional_access"
-)
+google_cloud_cpp_absl_pkg_config(absl_pkg_config google_cloud_cpp_common)
+string(CONCAT GOOGLE_CLOUD_CPP_PC_LIBS "-lgoogle_cloud_cpp_common"
```

Comment:
It is a little strange that we depend on the result of `google_cloud_cpp_absl_pkg_config` starting with a space.

Reply:
Yeah. Unfortunately `string(JOIN)` is not available until cmake-3.12, and we need to support cmake-3.5 (for now, we should review that).  I guess we could implement our own `google_cloud_cpp_join()` function, but I am lazy.